mobile-browser-based-version/src/helpers/memory/helpers.js

```diff
@@ -138,3 +138,45 @@ export async function downloadSavedModel(taskID, modelName) {
     DOWNLOADS_SCHEME.concat(`${taskID}_${modelName}`)
   );
 }
+/**
+ * Preprocesses the data based on the training information
+ * @param {Array} data the dataset of the task
+ * @param {Object} trainingInformation the training information of the task
+ */
+export function preprocessData(data, trainingInformation) {
+  var tensor = data;
+  //More preprocessing functions can be added using this template
+  if (trainingInformation.preprocessFunctions.includes('resize')) {
```

Comment:
To make this more generic do the following:

Create a function in helpers, just above this maybe and call it withImageResize(arg) (or something simple that scribes what it does), where arg is trainingInformation.preprocessFunctions, and it's output is true when we want to resize and image. This makes it more legible, but more importantly, as I note in my other [comment](https://github.com/epfml/DeAI/pull/182#issuecomment-1016254376), the resize might be redundant, and so an equality check might be better, in this case the readability gain is even bigger!